android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1178,11 +1225,17 @@ public void setFullscreen(boolean fullscreen) {
                 uiOptions = SYSTEM_UI_FLAG_HIDE_NAVIGATION
                         | SYSTEM_UI_FLAG_FULLSCREEN;
             }
+            activity.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
```

Comment:
@IbrahimSulai Could you set this to `SCREEN_ORIENTATION_SENSOR_LANDSCAPE` instead. So it can be either normal or reverse landscape.

Reply:
@r0b0t3d - Have updated the orientation to SCREEN_ORIENTATION_SENSOR_LANDSCAPE.

Please review and approve the PR.